features/test_runner/fastlane_test_runner.rb

```diff
@@ -42,8 +47,10 @@ def run
       # this only takes a few ms the first time being called
       Fastlane.load_actions
 
+      # TODO: How do I access fastlane actions (like lane_context) from here?
```

Comment:
You already have that in the `ensure` block, right?

Reply:
Yeah! Deleting it in next commit 😄